docs/readthedocs/source/_toc.yml

```diff
@@ -81,6 +81,7 @@ subtrees:
               title: "Key Features"
               subtrees:
                 - entries:
+                  - file: doc/Nano/Overview/cuda_patch
```

Comment:
I am not very sure whether we should put this CUDA patch as the first feature? Is it a feature for PyTorch only? Should we put it after PyTorch train & PyTorch inference, and also add "PyTorch" in its title somehow?